Fusion+Orc Exposures (#9123)
fishtown-analytics/docs.getdbt.com: website/docs/docs/platform-integrations/downstream-exposures-tableau.md

[Orchestrate exposures](/docs/platform-integrations/orchestrate-exposures) using the dbt [Cloud job scheduler](/docs/deploy/job-scheduler) to proactively refresh the underlying data sources (extracts) that power your Tableau Workbooks.

> This beta capability requires deployment jobs on [Latest](/docs/dbt-versions/cloud-release-tracks) with <Constant name="core" /> and is <em>not</em> available on [Latest&nbsp;<Constant name="fusion" />](/docs/dbt-versions/cloud-release-tracks) (the <Constant name="fusion_engine" /> preview). For setup details, refer to [Orchestrate exposures](/docs/platform-integrations/orchestrate-exposures).

Reviewer: ```suggestion
This beta capability requires deployment jobs on [Latest](/docs/dbt-versions/cloud-release-tracks) with <Constant name="core" /> and _isn't_ available on [Latest <Constant name="fusion" />](/docs/dbt-versions/cloud-release-tracks) (the <Constant name="fusion_engine" /> preview). For setup details, refer to [Orchestrate exposures](/docs/platform-integrations/orchestrate-exposures).
```
